pkg/errors/errors.go

```diff
@@ -1,31 +1,37 @@
-package linodego
+package errors
 
 import (
 	"fmt"
-	"log"
 	"net/http"
 	"strings"
 
 	"github.com/go-resty/resty/v2"
 )
 
 const (
-	// ErrorFromString is the Code identifying Errors created by string types
-	ErrorFromString = 1
-	// ErrorFromError is the Code identifying Errors created by error types
-	ErrorFromError = 2
-	// ErrorFromStringer is the Code identifying Errors created by fmt.Stringer types
-	ErrorFromStringer = 3
+	// FromString is the Code identifying Errors created by string types
+	FromString = iota + 1
```

Comment:
This change will need to be called out specifically, as everything else, people will only need to add `errors.` as a prefix, but this one is also a rename.

Reply:
When it comes time to cut the release, I'll be referencing what I called out in the above description.